com.lunarclient.LunarClient.appdata.xml

```diff
@@ -0,0 +1,1077 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+
+	<id>com.lunarclient.LunarClient</id>
+
+	<metadata_license>CC0-1.0</metadata_license>
+
+	<name>Lunar Client</name>
+
+	<summary>A modpack for all modern versions of Minecraft!</summary>
+
+	<description>
+		<p>Boosted frames, all the popular mods, multi OS support, and did we mention it’s completely free? So what are you waiting for - Download today! </p>
+	</description>
+
+	<categories>
+		<category>Game</category>
+	</categories>
+
+	<url type="homepage">https://lunarclient.com</url>
+	<url type="help">https://support.lunarclient.com</url>
+	<url type="faq">https://www.lunarclient.com/faq</url>
+
+	<launchable type="desktop-id">com.lunarclient.LunarClient.desktop</launchable>
+
+	<releases>
+		<release version="2.9.3" date="2021-12-03">
+			<url>https://www.lunarclient.com/news/patch-notes-17/</url>
+			<description>
+				<p>
+				Hey everyone, our primary focus these past few weeks has been to fix bugs, implement new mods, and add quality of life features!
+				</p>
+				
+				<p>
+				We finally have the much-requested Quickplay Mod. This mod allows you to switch games on the Hypixel Network seamlessly. 
+				We also added a new mod called “Snaplook” that will enable you to change/toggle your perspective by pressing a set key (no, this is not freelook). 
+				Lastly, we added the 3D Skins mod that allows you to add more details to your extra skin layers. 
+				Along with adding these new mods, we completely revamped the Crosshair Mod, fixed some much-needed bugs, and improved various aspects of our UI. 
+				Oh – we also have Discord linking! If you aren’t yet in our Discord, make sure to join!
+				</p>
+				
+	   			<p>
+	   			Also, we are aware of launching issues on Intel macOS on versions 1.7, 1.8, and 1.12. 
+	   			We are actively working on a solution and should have an update for this soon, sorry for the inconvenience.
+	   			</p>
+	   			
+	   			<p>
+	   			If you have any suggestions to improve Lunar Client, make sure to interact with us on our social media accounts (https://www.lunarclient.com/socials).
+	   			</p>
+	   			
+				<p>
+				We are still actively searching for new hires. Specifically, we are looking for new client developers to implement and create new Hypixel Skyblock mods. 
+				Visit our jobs page (https://www.lunarclient.com/jobs) for all relevant information. 
+				Thank you all for the constant support; Lunar Client is made for the players, by the players.
+				</p>
+ 
+			</description>
+		</release>
+		<release version="2.8.4" date="2021-9-26">
+			<url>https://www.lunarclient.com/news/patch-notes-15/</url>
+		</release>
+		<release version="2.7.5" date="2021-8-29">
+			<description>
+				<p>Weather changer</p>
+				<p>
+					First flatpak release
+				</p>
+			</description>
+		</release>
+	</releases>
+
+	<requires>
+		<control>keyboard</control>
+	</requires>
+
+	<project_license>Proprietary</project_license>
```

Comment:
should be `<project_license>LicenseRef-proprietary</project_license>`

Reply:
yes, see https://github.com/flathub/flathub/pull/2784#issuecomment-1018858368